hrui/CRM/HRUI/Helper.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Collection of methods to be used in this extension
+ */
+class CRM_HRUI_Helper {
+
+  /**
+   * @param int $contactID contact ID
+   * @return string|null return comma separated string containing list of line managers
```

Comment:
This is also a non-blocker, but I'm thinking that maybe it would be better to return an array instead of a comma separated string. Separating managers with commas it's more like a UI decision, so I don't think this method should be responsible for it.